app/Console/Commands/Swagger2ParseServices.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Exceptions\DataFormatException;
+use GuzzleHttp\Client;
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Storage;
+use Webpatser\Uuid\Uuid;
+
+class Swagger2ParseServices extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'gateway:parse2';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Parse Swagger 2 API documentation of services';
+
+    /**
+     * @var Client
+     */
+    protected $client;
+
+    /**
+     * @var array
+     */
+    protected $config;
+
+    /**
+     * ParseServices constructor.
+     * @param Client $client
+     * @param array $config
+     */
+    public function __construct(Client $client, $config = null)
+    {
+        parent::__construct();
+        $this->client = $client;
+        $this->config = $config ?: app('config')->get('gateway');
+    }
+
+    /**
+     * @return array
+     */
+    private function getActions()
+    {
+        $actions = [];
+        
+        foreach($this->config['services'] as $serviceId => $settings){
+            $this->info('** Parsing ' . $serviceId);
+
+            $docRoot = $settings['doc_point'] ?? $this->config['global']['doc_point'];
+            $hostname = $settings['hostname'] ?? $serviceId . '.' . $this->config['global']['domain'];
+            $url = $settings['url'] ?? 'http://' . $hostname;
+            $prefix = $settings['prefix'];
```

Comment:
Btw, as seen on this line I allow 'prefix' to be set in the api settings. This way you can decide per api what the prefix is